recipes/conda-devenv/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "conda-devenv" %}
+{% set version = "0.9.1" %}
+{% set hash = "fe8e9be7d4f2df03b9a269a4b4b36ca67477c9b1d5c2af1af440f63b787f2f05" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: '{{ version }}.tar.gz'
+  url: https://github.com/ESSS/conda-devenv/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+      - conda-devenv = conda_devenv.devenv:main  
```

Comment:
nit: Would be nice if the indent size is uniform (e.g. only 2 spaces more here than `entry_points`).